.github/dependabot.yml

```diff
@@ -10,3 +10,9 @@ updates:
         # Ignore version upgrades.
         # Security updates are nevertheless. unaffected by this setting and will continue to work.
         update-types: ["version-update:semver-patch", "version-update:semver-minor", "version-update:semver-major"]
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
Is it not too rare? Checking more often doesn't cost us anything?

Reply:
They sometimes do multi releases in the same day, I don’t want dependabot to make a million PRs per month and a monthly rate sounds good to keep an eye on it without it being too verbose. Wdyt?